llvm/test/CodeGen/AArch64/sve2p1-fixed-length-fdot.ll

```diff
@@ -0,0 +1,230 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 6
+; RUN: llc -mtriple=aarch64-linux-gnu -mattr=+sve2 < %s | FileCheck %s --check-prefixes=CHECK,SVE2
+; RUN: llc -global-isel -global-isel-abort=2 -mtriple=aarch64-linux-gnu -mattr=+sve2 < %s | FileCheck %s --check-prefixes=CHECK,SVE2
```

Comment:
Please remove this and the other global-isel RUN line because these tests specifically exist to verify the SelectionDAG based lowering and nothing else.